### ISSUE TITLE
EDITOR ON PC TUTORIAL

### ISSUE THREAD

**Issue by flaminiacav** [jfrohnhofen/metamaterial-mechanisms#8]
Hi, I think I could install the editor on my machine (into rhino) but i still can't figure out how it works.
I mean if I launch the 1-START-SIMULATION.bat it opens Rhino, and his grasshopper windows but does not appear nothing similar to the editor online.
I would be interested in simulating that seems to work only on pc editor that's why i am trying to figure it out..
thank you

**Comment by flaminiacav** [comment]
if i open grasshopper apparently there is the metamaterial
![Cattura2](https://user-images.githubusercontent.com/10087767/70100531-50eb5f80-1632-11ea-80ae-d5618f1efecf.JPG)

**Comment by flaminiacav** [comment]
but something is missing (if I click download and install it doesn't do anything)
and if i close and duble click metamatsimulation it opens nodes that i really cannot understand
![Cattura3](https://user-images.githubusercontent.com/10087767/70100597-7c6e4a00-1632-11ea-949d-75739b791f32.JPG) I am on rhino 5 windows 10

**Comment by alexiiion** [comment]
are all 4 files from this repository /simulation/bin copied to the grasshopers Components Folder folder? did you check if any are blocked by windows, as described in the instructions?

**Comment by flaminiacav** [comment]
yeah they are, i solved for the input output connection error, the point was that I had to unblock not only the two .dll files as the tutorial says  but also the other two grasshopper file themselves..
But I still can't make the simulation works...
but I have a question:
do I have to connect rhino's grasshopper with metamaterial editor via web socket?
(in this case the real point is that I have a lack of knowledge in this, in case do you have any link to sort of tutorials?)

thank's a lot for replying in any case!

**Comment by alexiiion** [comment]
yes, the editor communicates to grasshopper via websocket. everything is implemented. if both (editor and grasshopper) are running on the same computer, you have to check that the url is set as in the image below. that should be default case though. 

![image](https://user-images.githubusercontent.com/17621027/71251996-0cdfa680-2324-11ea-9479-2bacc2e74002.png)

can you open the chrome console and see what the output is there? open it with F12 or right-click "inspect", then choose the tab "console".

**Comment by flaminiacav** [comment]
now it works for that part, the thing is that the karamba is not longer available in the link they put!
i donwloaded the one in this page https://www.food4rhino.com/app/karamba3d but it gives me error anyway as you can see, does someone still have the old one?
(thanks a lot alexiiion!)
![Cattura](https://user-images.githubusercontent.com/10087767/73565763-91186400-4462-11ea-864e-a31bc8335335.PNG)

**Comment by flaminiacav** [comment]
Yey!
it finally worked but i still have a question!!!! 
_[but bythewaythanks a lot alexiiion (it was the url and also the installing of karamba that by default installed it in the wrong rhino folder (it put it into a self created folder called 5.0 rhino instead of the one called just 5 rhino i just had to change path while installing)]_
 just a curiosity, it seems not to be very realistic but maybe i am using it in the wrong way and i have to se something in grassopper! the thing is that as you see in the image below if I anchor an extreme of the geometry and i just push on the other extreme angle it does not gust bend (as i expected) but it twist as well! can i modified it in order just to make the geometry bending 
![Senza titolo-1](https://user-images.githubusercontent.com/10087767/73591010-a71d3780-44e9-11ea-8b5a-84e98320e29c.jpg)
